sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_client_async.py

```diff
@@ -216,6 +216,9 @@ def get_queue_receiver(self, queue_name: str, **kwargs: Any) -> ServiceBusReceiv
         :paramtype receive_mode: ~azure.servicebus.ReceiveMode
         :keyword Optional[float] max_wait_time: The timeout in seconds between received messages after which the
          receiver will automatically stop receiving. The default value is None, meaning no timeout.
+        :keyword Optional[AutoLockRenewer] auto_lock_renewer: An AutoLockRenewer can be provided such that messages are
```

Comment:
should we mention that for async, it should be `azure.servicebus.aio.AutoLockRenewer`.